### PR TITLE
Improve contributing guide #131

### PR DESCRIPTION
**Pull request by PitifulPete**
Rename TOC heading to contribute. Add prerequisite for contributing to docs. Include quality checks by listing GitHub actions.

### CODE REVIEW

File: contribute/landing.md [hunk 1]
```diff
@@ -9,9 +9,18 @@ We welcome contributions, suggestions, fixes and constructive feedback from the
...
+
 The navigational structure, style, and content of our documentation follows the Diátaxis systematic framework for technical documentation. This categorises the documentation into tutorials, how-to guides, reference material and explanatory text. 
 
-Consistency is vital in documentation, which is why we request contributors to follow the {ref}`style-guide`. However, do not let this be a barrier to your contribution. You can still submit contributions to the best of your ability, and if something is inconsistent, we will fix it. We are targeting continuous improvement rather than delayed perfection.
+Consistency is vital in documentation, which is why we request contributors to follow the {ref}`style-guide`. We also set up the following GitHub actions to run automatically on every pull request raised against our docs repository:
+
+- Inclusive language check 
+- Spellcheck
+- Accessibility check
+- Link check.
```
Comment (keirthana) [inline contribute/landing.md:21]:
```suggestion
- Link check
```

File: contribute/landing.md [hunk 2]
```diff
@@ -9,9 +9,18 @@ We welcome contributions, suggestions, fixes and constructive feedback from the
 - Report it as a bug on [Launchpad](https://bugs.launchpad.net/anbox-cloud/+bugs) or on [GitHub](https://github.com/canonical/anbox-cloud-docs/issues/new).
 - Talk to us on the [public Matrix channel](https://matrix.to/#/#anbox-cloud:ubuntu.com).
 
+Contributing to our documentation is straightforward. We only require that all contributors sign the [Canonical contributor license agreement](https://ubuntu.com/legal/contributors), have a GitHub account, and set up a [git environment](https://github.com/canonical/open-documentation-academy/blob/main/getting-started/get_started.md#getting-your-environment-set-up).
```
Comment (keirthana) [inline contribute/landing.md:12]:
```suggestion
Contributing to our documentation is straightforward. We only require that all contributors sign the [Canonical contributor license agreement](https://ubuntu.com/legal/contributors).

In order to contribute, you will need a [GitHub](https://github.com/) account, and set up a [git environment](https://github.com/canonical/open-documentation-academy/blob/main/getting-started/install_git.md). The [Get Started guide](https://github.com/canonical/open-documentation-academy/blob/main/getting-started/get_started.md#getting-your-environment-set-up) from the Canonical Open Documentation Academy is a useful resource.
```

**Review by keirthana — CHANGES_REQUESTED** [review]
Thank you so much for trying to make the contributing guide better @PitifulPete! I have made some minor suggestions.

Also, let me know if you have signed the CLA (currently, the CLA check fails, but it does not pick up recent signups immediately)

**Comment by PitifulPete** [comment]
> Thank you so much for trying to make the contributing guide better @PitifulPete! I have made some minor suggestions.
> 
> Also, let me know if you have signed the CLA (currently, the CLA check fails, but it does not pick up recent signups immediately)

I have signed the CLA

**Comment by keirthana** [comment]
> > Thank you so much for trying to make the contributing guide better @PitifulPete! I have made some minor suggestions.
> > Also, let me know if you have signed the CLA (currently, the CLA check fails, but it does not pick up recent signups immediately)
> 
> I have signed the CLA

@PitifulPete It looks like the email you have used for this contribution does not match with the email with your Launchpad account? Could you check? That could be the reason why the CLA check is failing.

**Comment by PitifulPete** [comment]
Hi @keirthana, you were right about the email discrepancy. I have fixed it by amending the last commit with a consistent email address but the CLA check still failed with a new error stating that I haven't signed it at all even though I have

**Comment by keirthana** [comment]
> Hi @keirthana, you were right about the email discrepancy. I have fixed it by amending the last commit with a consistent email address but the CLA check still failed with a new error stating that I haven't signed it at all even though I have

@PitifulPete Unfortunately, your commit messages still have the email that is not connected to the LP account. Looking at the errors, it feels like the email with which you signed the CLA is not the email that is part of all your commits.

**Comment by PitifulPete** [comment]
Solved. You can merge now @keirthana

**Review by keirthana — APPROVED** [review]

**Pull request merged**